src/WebJobs.Script.WebHost/Global.asax.cs

```diff
@@ -11,6 +11,13 @@ public class WebApiApplication : System.Web.HttpApplication
         protected void Application_Start()
         {
             GlobalConfiguration.Configure(WebApiConfig.Register);
+
+            var scriptHostManager = GlobalConfiguration.Configuration.DependencyResolver.GetService<WebScriptHostManager>();
+
+            if (scriptHostManager != null && !scriptHostManager.Initialized)
```

Comment:
How does this play with the Initialize call in `WebScriptHostHandler`? I think this will lead to double init?